x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -858,13 +851,20 @@ export const IMPORT_FAILED_DETAILED = (message: string) =>
     }
   );
 
-export const DELETE_CONFIRMATION_TITLE = i18n.translate(
-  'xpack.securitySolution.detectionEngine.components.allRules.deleteConfirmationTitle',
+export const BULK_DELETE_CONFIRMATION_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.components.allRules.bulkDeleteConfirmationTitle',
   {
     defaultMessage: 'Confirm bulk deletion',
   }
 );
 
+export const SINGLE_DELETE_CONFIRMATION_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.components.allRules.singleDeleteConfirmationTitle',
+  {
+    defaultMessage: 'Confirm deletion',
+  }
+);
+
 export const DELETE_CONFIRMATION_CONFIRM = i18n.translate(
```

Comment:
nit: The confirm button text should match the action, so in this instance it should be `Delete`.

https://elastic.github.io/eui/#/layout/modal/guidelines

<img width="1259" alt="image" src="https://github.com/elastic/kibana/assets/2946766/4a04ef04-43a1-4972-a2ed-788cef30d65c">